src/main/scala/org/globalforestwatch/summarystats/gladalerts/GladAlertsSummary.scala

```diff
@@ -10,32 +10,32 @@ import org.globalforestwatch.util.{Geodesy, Mercantile}
 import scala.annotation.tailrec
 
 /** LossData Summary by year */
-case class GladAlertsSummary(stats: Map[GladAlertsDataGroup, GladAlertsData] =
-                             Map.empty,
-                             kwargs: Map[String, Any])
+case class GladAlertsSummary(stats: Map[GladAlertsDataGroup, GladAlertsData] = Map.empty)
   extends Summary[GladAlertsSummary] {
 
   /** Combine two Maps and combine their LossData when a year is present in both */
   def merge(other: GladAlertsSummary): GladAlertsSummary = {
     // the years.combine method uses LossData.lossDataSemigroup instance to perform per value combine on the map
-    GladAlertsSummary(stats.combine(other.stats), kwargs)
+    GladAlertsSummary(stats.combine(other.stats))
   }
 }
 
 object GladAlertsSummary {
   // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
 
   implicit val mdhCellRegisterForTreeLossRaster1
-    : CellVisitor[Raster[GladAlertsTile], GladAlertsSummary] =
-    new CellVisitor[Raster[GladAlertsTile], GladAlertsSummary] {
+    : GridVisitor[Raster[GladAlertsTile], GladAlertsSummary] =
+      new GridVisitor[Raster[GladAlertsTile], GladAlertsSummary] {
+      private var acc: GladAlertsSummary = new GladAlertsSummary()
 
-      def register(raster: Raster[GladAlertsTile],
+      def result: GladAlertsSummary = acc
+
+      def visit(raster: Raster[GladAlertsTile],
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)